django/db/migrations/operations/fields.py

```diff
@@ -70,7 +70,8 @@ def state_forwards(self, app_label, state):
         else:
             field = self.field
         state.models[app_label, self.model_name_lower].fields.append((self.name, field))
-        state.reload_model(app_label, self.model_name_lower)
+        delay = not field.is_relation  # Delay rendering of relationships if it's not a relational field
```

Comment:
Considering our guideline is to wrap comments at 79, I'd put these comments above the line rather than next to it.